kraken/network_scenarios/actions.py

```diff
@@ -0,0 +1,83 @@
+import yaml
+import logging
+import time
+import kraken.cerberus.setup as cerberus
+import kraken.kubernetes.client as kubecli
+import kraken.node_actions.common_node_functions as common_node_functions
+
+
+pod_yml = """
+apiVersion: v1
+kind: Pod
+metadata:
+  name: fedtools
```

Comment:
Maybe change the name to something along the lines of kraken-network-chaos? Thoughts?

Reply:
yes sounds good,  how about "network-chaos", since prefixing kraken would be kind of redundant.

Reply:
Sounds good!